hw3/hw3-makinunal/hw3-akinunal.js

```diff
@@ -0,0 +1,131 @@
+/* 1. write a constructor function PersonFunc which accepts the person data like in file people.data.js and implements next methods:
+getFullName() => returns string in format "[FirstName] [LastName]", e.g. 'Viktoriia Chernyshuk';
+getAge() => returns age of person in format "[number] years", e.g. "25 years";
+getFullAddress() => returns address string in format "[country], [city], [street], [house]/[apartment]". Field apartment is optional. E.g. "Ukraine, Lviv, Doroshenka street, 5/8" and "Ukraine, Lviv, Muchna street, 7".
+Please implement error handling in case if some fields are not available.
+*/
+import { SarahMay, JeremieBrown, Elizabeth } from './people.data.js';
```

Comment:
why did you create the same file as you had in a folder `hw3`. You could import those consts like `import { SarahMay, JeremieBrown, Elizabeth } from '../people.data.js'`. Or it didn't work that way?

Reply:
When i start working on homeworks, i copy the necessary files into a folder. And when i am done with the homework, i copy them into my branch folder in repo. I believe "from '../people.data.js'" could work for git repo but i was working on it when the files were in same folder.

Reply:
could you try if import `{ SarahMay, JeremieBrown, Elizabeth } from '../people.data.js'` works? Because having dozen of the same files is not a good thing if there is no serious reason for that.